tcms/templates/include/bugs_table.html

```diff
@@ -0,0 +1,29 @@
+{% load i18n %}
+{% load static %}
+
+<div class="col-xs-12 col-sm-6 col-md-4">
+    <div class="card-pf card-pf-accented">
+        <h2 class="card-pf-title">
+            {% if show_hyperlinks %}
+                {% trans 'Bugs and Hyperlinks' %}
+            {% else %}
+                <span class="fa fa-bug"></span>
+                {% trans 'Bugs' %}
+            {% endif %}
```

Comment:
Pass the card title as variable to include. Icon is fine to be the same for now.